src/main/java/nl/tudelft/semgroup4/StartScreenState.java

```diff
@@ -1,183 +1,182 @@
-                multiplayerGame.getCurLevel().toAdd(firstPlayer.getWeapon());
-
-                final GameState gameState = new GameState(multiplayerGame);
-
-                game.addState(gameState);
-                game.getState(States.GameState).init(container, game);
-                resources.stopTitleScreen();
-                game.enterState(States.GameState);
-                Game.LOGGER.log(LogSeverity.DEBUG, "StartMenu",
-                        "User starts a multi-player game");
-            } else if (mouseOverOptions.isMouseOver()) {
-                input.clearKeyPressedRecord();
-                input.clearMousePressedRecord();
-                game.enterState(States.OptionsState);
-                Game.LOGGER.log(LogSeverity.DEBUG, "StartMenu", "User enters options menu");
-            } else if (mouseOverQuit.isMouseOver()) {
-                Game.LOGGER.log(LogSeverity.DEBUG, "StartMenu", "User quits the application");
-                resources.stopTitleScreen();
-                container.exit();
-            }
+        updatePlayer1Button(container, game);
+        updatePlayer2Button(container, game);
+        updateOptionsButton(container, game);
+        updateExitButton(container, game);
+        updateHighscoresButton(container, game);
+    }
+
+    /**
+     * Checks if the High Scores button is pressed and performs all necessary actions if it is.
+     * 
+     * @param container
+     *            {@link GameContainer} - The Slick2D GameContainer.
+     * @param game
+     *            {@link StateBasedGame} - The Slick2D StateBasedGame main class.
+     * @throws SlickException
+     *             If switching states causes an exception.
+     */
+    protected void updateHighscoresButton(GameContainer container, StateBasedGame game)
+            throws SlickException {
+        if (controller.isAreaClicked(mouseOverHighScores, input)) {
+            game.getState(States.HighscoresState).init(container, game);
+            game.enterState(States.HighscoresState);
+            Game.LOGGER.log(LogSeverity.DEBUG, "StartMenu", "User enters Highscores menu");
         }
+    }
 
+    /**
+     * Checks if the 1 Player button is pressed and performs all necessary actions if it is.
+     * 
+     * @param container
+     *            {@link GameContainer} - The Slick2D GameContainer.
+     * @param game
+     *            {@link StateBasedGame} - The Slick2D StateBasedGame main class.
+     * @throws SlickException
+     *             If switching states causes an exception.
+     */
+    protected void updatePlayer1Button(GameContainer container, StateBasedGame game)
+            throws SlickException {
+        if (controller.isAreaClicked(mouseOverOnePlayer, input)) {
+            Game singleplayerGame = controller.createSingleplayerGame(container, game);
+
+            final GameState gameState = new GameState(singleplayerGame);
+
+            game.addState(gameState);
```

Comment:
line 109 till 112 are the same in both updatePlayer1Button and updatePlayer2Button and can be put in a different method.